src/SAWScript/Prover/Exporter.hs

```diff
@@ -6,6 +6,8 @@
 {-# Language ExplicitForAll #-}
 {-# Language FlexibleContexts #-}
 {-# Language TypeApplications #-}
+{-# Language ParallelListComp #-}
```

Comment:
Note to self: these aren't used anymore, so we don't need this language extension.